{"uuid": "915c5b36-d94c-4399-a202-60b064c06071", "children": ["5d922bb8-86ee-4479-a5ac-b5a89d527bcb"], "befores": [{"name": "setupbrowser", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"C:\\Users\\Dell\\PycharmProjectHRMS.PYTEST\\pythonProject\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dell\\PycharmProjectHRMS.PYTEST\\pythonProject\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\Dell\\PycharmProjectHRMS.PYTEST\\pythonProject\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\Dell\\PycharmProjectHRMS.PYTEST\\pythonProject\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\Dell\\PycharmProjectHRMS.PYTEST\\pythonProject\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\Dell\\PycharmProjectHRMS.PYTEST\\pythonProject\\tests\\conftest.py\", line 39, in setupbrowser\n    driver = webdriver.Chrome(options=chrome_options)\n  File \"C:\\Users\\Dell\\PycharmProjectHRMS.PYTEST\\pythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 45, in __init__\n    super().__init__(\n    ~~~~~~~~~~~~~~~~^\n        browser_name=DesiredCapabilities.CHROME[\"browserName\"],\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    ...<3 lines>...\n        keep_alive=keep_alive,\n        ^^^^^^^^^^^^^^^^^^^^^^\n    )\n    ^\n  File \"C:\\Users\\Dell\\PycharmProjectHRMS.PYTEST\\pythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 55, in __init__\n    self.service.start()\n    ~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\Dell\\PycharmProjectHRMS.PYTEST\\pythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 109, in start\n    if self.is_connectable():\n       ~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\Dell\\PycharmProjectHRMS.PYTEST\\pythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 126, in is_connectable\n    return utils.is_connectable(self.port)\n           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\Users\\Dell\\PycharmProjectHRMS.PYTEST\\pythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\common\\utils.py\", line 101, in is_connectable\n    socket_ = socket.create_connection((host, port), 1)\n  File \"C:\\Program Files\\JetBrains\\PyCharm Community Edition 2024.2.4\\plugins\\python-ce\\helpers\\pydev\\_pydevd_bundle\\pydevd_pep_669_tracing.py\", line 542, in py_raise_callback\n    def py_raise_callback(code, instruction_offset, exception):\n    \n"}, "start": 1739860742270, "stop": 1739860743730}], "start": 1739860742270, "stop": 1739860745284}